ipa-core/src/report/hybrid.rs

```diff
@@ -224,35 +363,132 @@ impl<BK, V> HybridReport<BK, V>
 where
     BK: SharedValue,
     V: SharedValue,
+    Replicated<BK>: Serializable,
+    Replicated<V>: Serializable,
+    <Replicated<BK> as Serializable>::Size: Add<U16>,
+    <Replicated<V> as Serializable>::Size: Add<U16>,
+    <<Replicated<BK> as Serializable>::Size as Add<<Replicated<BA64> as Serializable>::Size>>:: Output: ArrayLength,
+    <<Replicated<V> as Serializable>::Size as Add<<Replicated<BA64> as Serializable>::Size>>:: Output: ArrayLength,
 {
+    /// # Panics
+    /// If report length does not fit in `u16`.
+    pub fn encrypted_len(&self) -> u16 {
```

Comment:
It seems Codecov wants some coverage for the impression path as well

Reply:
Yeah, fixing the query runner tests to use both impressions and conversions should help here